django/core/management/templates.py

```diff
@@ -104,6 +106,7 @@ def handle(self, app_or_project, name, target=None, **options):
             base_name: name,
             base_directory: top_dir,
             camel_case_name: camel_case_value,
+            'app_path': app_path,
```

Comment:
I would add `python_path` for both (for consistency), e.g.
```python
base_python_path = '%s_python_path' % app_or_project
...
context = Context({
    **options,
    base_python_path: python_path,
    base_name: name,
    base_directory: top_dir,
    camel_case_name: camel_case_value,
    'docs_version': get_docs_version(),
    'django_version': django.__version__,
}, autoescape=False)
```
also `app_python_path` and `project_python_path` (always equal to `project_name`) should be added to the [documentation](https://docs.djangoproject.com/en/2.2/ref/django-admin/) with `versionadded` admonition and to the release notes (in section `Minor features -> Management Commands`).